composer.json

```diff
@@ -134,8 +137,10 @@
         "su-sws/stanford_person": "dev-8.x-1.x",
         "su-sws/stanford_profile_helper": "dev-8.x-1.x",
         "su-sws/stanford_ssp": "dev-8.x-1.x",
-        "su-sws/stanford_text_editor": "dev-8.x-1.x",
-        "drupal/menu_block": "dev-1.x#567becffbb0589e824fb053f15fb38a5846e7276"
+        "su-sws/stanford_text_editor": "dev-8.x-1.x"
+    },
+    "config": {
+        "sort-packages": true
```

Comment:
Cool.

Reply:
it helps when doing `composer require [package] --no-update`. less to worry about.